### ISSUE TITLE
bug: null is not an object (evaluating 'e.shadowRoot')

### ISSUE THREAD

**Issue by nvahalik** [ionic-team/ionic-framework#22751]
# Bug Report

**Ionic version:**
[ ] **4.x**
[x] **5.x**

**Current behavior:**
When pulling on an `<ion-refresher>` element Safari/WebKit will throw the following error: 

    null is not an object (evaluating 'e.shadowRoot')

The element also seems to not properly display. It seems to skip and not expand/retract.

**Expected behavior:**
It should function how it normally does. This seems to have primarily been an issue when upgrading from 5.4.1 to 5.5.2.

**Steps to reproduce:**
Well, usually someone just pulls down on an ion-refresher in the app. Only appears to affect WebKit? I cannot reproduce on Firefox.

**Related code:**
Here is the trace:

```
File http://app/105-es2015.26880b8766da98a02540.js  line 1 col 3967 in [anonymous]
setupiOSNativeRefresher()
File [native code] line (unknown) in asyncFunctionResume
File http://app/105-es2015.26880b8766da98a02540.js  line 1 col 8602 in [anonymous]
setupNativeRefresher()
File [native code] line (unknown) in asyncFunctionResume
File http://app/105-es2015.26880b8766da98a02540.js  line 1 col 3336 in [anonymous]
checkNativeRefresher()
File [native code] line (unknown) in asyncFunctionResume
File [native code] line (unknown) in promiseReactionJobWithoutPromise
```

I have tried to reproduce and sometimes I can. Sometimes I cannot. Perhaps someone else can reproduce this more reliably.

It doesn't appear to cause any major functionality issues, other than the fact that it just abruptly closes and then starts the refresh cycle.

**Other information:**
As mentioned before, this only started happening after an update from 5.4.1 to 5.5.2. Looks like these changes: https://github.com/ionic-team/ionic-framework/commit/5ed73cdf4d63eeee25ef28d9676fcaa4f8e07b47#diff-b29397aad7f5dbb9aef05f5ce05a3bc7df6ae3858de670c6c4dcca0db23bc6a5 may be related?

**Ionic info:**

```
Ionic:

   Ionic CLI                     : 6.12.0 (/usr/local/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 5.5.2
   @angular-devkit/build-angular : 0.901.13
   @angular-devkit/schematics    : 10.2.1
   @angular/cli                  : 9.1.13
   @ionic/angular-toolkit        : 2.3.3

Cordova:

   Cordova CLI       : 9.0.0 (cordova-lib@9.0.1)
   Cordova Platforms : not available
   Cordova Plugins   : not available

Utility:

   cordova-res                          : 0.15.2
   native-run (update available: 1.3.0) : 0.2.8

System:

   ios-deploy : 1.9.4
   ios-sim    : 8.0.2
   NodeJS     : v14.5.0 (/usr/local/Cellar/node/14.5.0/bin/node)
   npm        : 6.14.5
   OS         : macOS Big Sur
   Xcode      : Xcode 12.3 Build version 12C33
```

**Comment by liamdebeasi** [comment]
Thanks for the issue. Can you reproduce this issue in an Ionic starter app and provide a link to the repo? I cannot reproduce this on my end.

**Comment by nvahalik** [comment]
OK. I think I have found the issue. It will be a few days before the updates trickle down, but the issue appears to be this:

The `pullingIcon` value was invalid. It was functional but it looked bad.

At any rate, we'll remove the invalid `pullingIcon` values and try it again.

**Comment by hoi4** [comment]
For me this error occurred when updating @ionic/angular from version 5.5.1 to version 5.5.2.

I am using a correct name of an ionicon ("arrow-down-circle-outline") and it fails for me. 🤔

**Comment by liamdebeasi** [comment]
@hoi4 Can you reproduce this in an Ionic starter app and provide a link to the repo?

**Comment by hoi4** [comment]
I created a repo: https://github.com/hoi4/ionic-refresher-issue

When the dev server starts, I enter the Chrome devtools and refresh --> Error occurs
After refreshing the page, I could not reproduce the issue.

I also had this issue on a device, so I don't think that the resize is the problem 🤔 

I created a small screen capture to illustrate the issue:
https://user-images.githubusercontent.com/23243256/104484754-f14bdb80-55c9-11eb-91d5-b4794336c368.mov

I hope it helps. Thank you!

**Comment by phal0r** [comment]
We saw this error popping up in Sentry aswell, when updating to 5.5.2 and using @ionic/react. It throws on Android and iOS devices. I didn't analyze it further, yet, but maybe this information helps to properly categorize it.

**Comment by nvahalik** [comment]
Using the report provided by @hoi4, I can indeed reproduce the issue and confirm that downgrading to 5.5.1 fixes it:

Here is what it does with 5.5.1:
![CleanShot 2021-01-28 at 09 38 48](https://user-images.githubusercontent.com/632120/106161688-eb6b0400-614c-11eb-8e5c-2d5d48238c88.gif)

Here is the same screen on 5.5.2:
![CleanShot 2021-01-28 at 09 40 06](https://user-images.githubusercontent.com/632120/106161733-f4f46c00-614c-11eb-89d6-67c240956ce9.gif)

**Comment by liamdebeasi** [comment]
Hi everyone,

This issue was fixed in https://github.com/ionic-team/ionic-framework/pull/22800 and a fix will be available when Ionic Framework v5.5.3 comes out later today.
